MonoGameGum/Forms/FormsUtilities.cs

```diff
@@ -45,10 +46,13 @@ public static void InitializeDefaults(Game? game = null, SystemManagers? systemM
                 "You must call this method after initializing SystemManagers.Default, or you must explicitly specify a SystemsManager instance");
         }
 
-        TryAdd(typeof(Button), typeof(DefaultButtonRuntime));
+        // Get the UI Spite Sheet that we loaded in SystemManager.Initialize()
+        var uiSpriteSheetTexture2D = RenderingLibrary.Content.LoaderManager.Self.GetDisposable("UISpriteSheet.png");
```

Comment:
I wanted to "pass this down" into the Visuals, but I don't think that's possible?

Also, the Changes in the TypeOf here are TEMPORARY, i need to change them back once I get a METHOD that allows me to change to V2.

Reply:
What you have here is fine for the texture. No need to pass it down.

Sounds good on the typeof changes.